<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoEqualizer2</class>
 <widget class="QWidget" name="VideoEqualizer2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Equalizer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Contrast</string>
       </property>
       <property name="buddy">
        <cstring>contrast_slider</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="contrast_slider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="contrast_label">
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Brightness</string>
       </property>
       <property name="buddy">
        <cstring>brightness_slider</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="brightness_slider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="brightness_label">
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Hue</string>
       </property>
       <property name="buddy">
        <cstring>hue_slider</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="hue_slider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="hue_label">
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Saturation</string>
       </property>
       <property name="buddy">
        <cstring>saturation_slider</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="saturation_slider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="saturation_label">
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&amp;Gamma</string>
       </property>
       <property name="buddy">
        <cstring>gamma_slider</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="gamma_slider">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="gamma_label">
       <property name="text">
        <string>-100</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="makedefault_button">
       <property name="text">
        <string>Set as &amp;default values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset_button">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
